### ISSUE TITLE
Incorrectly generated default value NULL as string

### ISSUE THREAD

**Issue by sanderboom** [Kononnable/typeorm-model-generator#153]
First of all: thanks for the great software.

Second: Generated entities contain the string `'NULL'` as `default` property. This is incorrect. It seems that the `default` property can be removed and results in a correctly generated schema.

I have the following table in a MariaDB 10.2.14 database:

```sql
CREATE TABLE `typeorm_test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `descr` varchar(1) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
````

`typeorm-model-generator` generates:

```typescript
import {BaseEntity,Column,Entity,Index,JoinColumn,JoinTable,ManyToMany,ManyToOne,OneToMany,OneToOne,PrimaryColumn,PrimaryGeneratedColumn,RelationId} from "typeorm";


@Entity("typeorm_test",{schema:"app_db"})
export class typeorm_test {

    @PrimaryGeneratedColumn({
        type:"int", 
        name:"id"
        })
    id:number;
        

    @Column("varchar",{ 
        nullable:true,
        length:1,
        default: () => "'NULL'",
        name:"descr"
        })
    descr:string | null;
        
}
```

Synchronizing the entity to the database gives:
```
query failed: CREATE TABLE `typeorm_test` (`id` int NOT NULL AUTO_INCREMENT, `descr` varchar(1) NULL DEFAULT 'NULL', PRIMARY KEY (`id`)) ENGINE=InnoDB
error: { Error: ER_INVALID_DEFAULT: Invalid default value for 'descr'
```

As `descr` is `VARCHAR(1)` the default value will not fit. More importantly, it should not be a string in the first place.

Removing `default: () => "'NULL'",` from the entity, creates the following (correct) table:
```sql
CREATE TABLE `typeorm_test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `descr` varchar(1) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Thanks for looking into this.

**Comment by sanderboom** [comment]
Also possible:

```typescript
@Column({ default: () => "NULL" })
```
Which is probably more consistent (but does not result in a different schema (in MariaDB 10.2.14 at least). See also: [TypeOrm FAQ](https://github.com/typeorm/typeorm/blob/master/docs/faq.md#how-can-i-set-the-default-value-to-some-function-for-example-now)

**Comment by sanderboom** [comment]
:raised_hands:

**Comment by Kononnable** [comment]
Fixed and released in `typeorm-model-generator@0.3.2`
